src/pages/access_review/AccessReviewHeader.js

```diff
@@ -16,10 +17,14 @@ const SMALL = {
 };
 
 export const AccessReviewHeader = hh(class AccessReviewHeader extends React.PureComponent {
-
-  openAccessReview = (referenceId) => {
```

Comment:
unused

Reply:
The functionality that this added was to ensure that the history was correctly populated. If not necessary anymore, we should remove `history` from all calls.